bin/cron/activity-monitor

```diff
@@ -200,7 +184,7 @@ def main
     level_url = "https://studio.code.org/levels/#{i[:level]}"
 
     log_string = %[<b>#{i[:browser]}</b> hasn't passed <b><a href="#{level_url}">level #{i[:level]}</a></b> despite <b>#{i[:starts]}</b> attempts in the last #{HOURS_TO_TEST} hour(s)]
-    log_string += %[ <a href="#{browserstack_url(i[:browser], level_url) || level_url}">✈ Repro It</a>]
```

Comment:
Note that `saucelabs_url`, unlike `browerstack_url` will never return `nil`. So the `level_url` disjunction is unnecessary.